jsx/typings/react/react.d.ts

```diff
@@ -0,0 +1,936 @@
+// Type definitions for React v0.13.1 (external module)
+// Project: http://facebook.github.io/react/
+// Definitions by: Asana <https://asana.com>, AssureSign <http://www.assuresign.com>, Microsoft <https://microsoft.com>
+// Definitions: https://github.com/borisyankov/DefinitelyTyped
+
+declare module __React {
```

Comment:
My understanding is that you shouldn't be checking in your `.d.ts` files in because `tsd` will take care of this for you